styles/style.css

```diff
@@ -1,22 +1,24 @@
-
 .page{
   margin: 0 auto;
+  padding: 0;
   width: 100%;
   font-family: 'Helvetica Neue', Arial, sans-serif;
   min-width: 1100px;
   max-width: 1600px;
 }
 
 .header{
+  margin: 0 auto;
+  padding: 0;
   display: flex;
   align-items: center;
   height: 100vh;
```

Comment:
Please, double-check the project brief for header height. Along with `height: 100vh;` we have to set `min-height` and `max-height` so the container doesn't shrink or stretch too much.